tests/load/locustfile.py

```
@@ -0,0 +1,56 @@
+from locust import HttpUser, task, between
+
+API_KEY = "test-api-key"
+
+
+class TENETLoadUser(HttpUser):
+    wait_time = between(1, 3)
+
+    headers = {
+        "x-api-key": API_KEY,
+        "Content-Type": "application/json",
+    }
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)